src/helpers/pivot/spreadsheet_pivot/spreadsheet_pivot.ts

```diff
@@ -224,10 +225,23 @@ export class SpreadsheetPivot implements Pivot<SpreadsheetPivotRuntimeDefinition
     if (!operator) {
       throw new Error(`Aggregator ${aggregator} does not exist`);
     }
-    return {
-      value: values.length ? operator.fn([values], this.getters.getLocale()) : "",
-      format: operator.format(values[0]),
-    };
+
+    try {
+      return {
+        value: values.length ? operator.fn([values], this.getters.getLocale()) : "",
+        format: operator.format(values[0]),
+      };
+    } catch (e) {
+      if (!(e instanceof EvaluationError)) {
+        throw e;
+      }
+      const errorFPayload = {
+        value: e.value ? e.value : CellErrorType.GenericError,
+        message: e.message,
+      };
+      replaceFunctionNamePlaceholder(errorFPayload, aggregator.toUpperCase());
+      return errorFPayload;
+    }
```

Comment:
would it work to export and reuse `handleError` ?
```suggestion
    } catch (e) {
      return handleError(e, aggregator.toUpperCase();
    }
```

Reply:
hurray then ^^

Reply:
Didn't see the helper, it sounds like a good idea :D